blocks/feed/feed.js

```diff
@@ -53,13 +54,22 @@ const resultParsers = {
  */
 export default async function decorate(block) {
   const blockCfg = readBlockConfig(block);
-  const blockType = blockCfg['block-type'].trim().toLowerCase();
+  const blockType = blockCfg['block-type'] ? blockCfg['block-type'].trim().toLowerCase() : 'cards';
   const queryObj = await queryIndex();
 
-  const results = queryObj.where((el) => (!blockCfg.type || blockCfg.type === el.type)
-    && (!blockCfg.category || blockCfg.category === el.category)
-    && (!blockCfg.modifier || el.modifier === blockCfg.modifier))
-    .orderByDescending((el) => (blockCfg.sort ? el[blockCfg.sort] : el.path)).take(blockCfg.count)
+  const type = blockCfg.type ? blockCfg.type.trim().toLowerCase() : getMetadata('type')?.trim().toLowerCase();
+  const category = blockCfg.category ? blockCfg.category.trim().toLowerCase() : getMetadata('category')?.trim().toLowerCase();
+  // eslint-disable-next-line prefer-arrow-callback
+  const results = queryObj.where(function filterElements(el) {
+    const elType = el.type ? el.type.trim().toLowerCase() : '';
+    const elCategory = el.category ? el.category.trim().toLowerCase() : '';
+    const elFeatured = el.featured ? el.featured.trim().toLowerCase() : '';
+    return (!type || type === elType)
+      && (!category || category === elCategory)
+      && (!blockCfg.featured || elFeatured === blockCfg.featured.trim().toLowerCase());
+  })
+    .orderByDescending((el) => (blockCfg.sort ? el[blockCfg.sort.trim().toLowerCase()] : el.path))
```

Comment:
> Nit: el[blockCfg.sort?.trim().toLowerCase()] ?? el.path

This may be problematic if `blockCfg.sort` is `null` as it translates to `el[null] ?? el.path`